server/admin/prompt.go

```diff
@@ -10,7 +10,7 @@ import (
 	"os"
 
 	"decred.org/dcrdex/dex/encode"
-	"golang.org/x/crypto/ssh/terminal"
```

Comment:
deprecated in https://github.com/golang/crypto/commit/4be66e5b658251a93e17d931a68d9c0ecba9f83a